internal/services/cdn/cdn_frontdoor_custom_domain_resource.go

```diff
@@ -268,21 +225,11 @@ func resourceCdnFrontDoorCustomDomainUpdate(d *pluginsdk.ResourceData, meta inte
 	ctx, cancel := timeouts.ForUpdate(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorCustomDomainID(d.Id())
+	id, err := parse.FrontDoorCustomDomainIDInsensitively(d.Id())
```

Comment:
this is being parsed from the Resource ID and so should be case-sensitive:

```suggestion
	id, err := parse.FrontDoorCustomDomainID(d.Id())
```